styles.css

```diff
@@ -29,14 +19,27 @@ body {
   justify-content: space-evenly;
   width: 100vw;
   height: 100vh;
-  background: linear-gradient(var(--primary), var(--secondary), var(--third));
+  /* background: linear-gradient(var(--primary), var(--secondary), var(--third)); */
+}
+
+.body-blue-to-yellow-bg {
+    background: linear-gradient(var(--primary), var(--secondary), var(--third));
```

Comment:
decrease indent here

Reply:
Changes made with indentations, pushed product up for review. Looked through entire comp to try and see if there were anymore that were missed but didn't see any.